code_samples/api/public_php_api/src/Command/TaxonomyCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Contracts\Taxonomy\Service\TaxonomyServiceInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class TaxonomyCommand extends Command
+{
+    private TaxonomyServiceInterface $taxonomyService;
+
+    private PermissionResolver $permissionResolver;
+
+    private UserService $userService;
+
+    public function __construct(TaxonomyServiceInterface $taxonomyService, PermissionResolver $permissionResolver, UserService $userService)
+    {
+        $this->taxonomyService = $taxonomyService;
+        $this->permissionResolver = $permissionResolver;
+        $this->userService = $userService;
+        parent::__construct('doc:taxonomy');
+    }
+
+    public function configure(): void
+    {
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $user = $this->userService->loadUserByLogin('admin');
+        $this->permissionResolver->setCurrentUserReference($user);
+
+        $allEntries = $this->taxonomyService->loadAllEntries('tags');
```

Comment:
There are optional (but important) `$limit` and `$offset` arguments that are worth pointing out:
```suggestion
        $allEntries = $this->taxonomyService->loadAllEntries('tags', 30, 0);
```